app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -133,8 +133,15 @@ function DocumentCapture({ onStepChange = () => {} }: DocumentCaptureProps) {
         },
       ].filter(Boolean) as FormStep[]);
 
+  // If the user got here by opting-in to in-person proofing, when skipDocAuth === 'true',
+  // then set steps to inPersonSteps
+  const steps: FormStep[] = skipDocAuth === 'true' ? inPersonSteps : defaultSteps;
+
+  // If the user got here by opting-in to in-person proofing, when skipDocAuth === 'true',
+  // then set stepIndicatorPath to VerifyFlowPath.IN_PERSON
```

Comment:
is this a user who failed the id upload step?

Reply:
No. This user picked IPP when given the chose.  If the user picked remote- (and opted out of IPP initially) the value of skipDocAuth is step to false. They will go through the remote flow and if they fail the id upload step- they will move into IPP but the value for skipDocAuth will still be false.  If you look above in this same folder, currently line 108, there is default steps- we concat inPersonSteps on 127- so you can still hit those.